compile error

Suddenly got this error, seems related to the new release. Note that I'm not using this crate directly. Any ideas?  
 
   Compiling flate2 v1.0.10
error[E0658]: use of unstable library feature 'rustc_private': this crate is being loaded from the sysroot, an unstable location; did you mean to load this crate from crates.io via `Cargo.toml` instead?
  --> /home/rust/.cargo/registry/src/github.com-1ecc6299db9ec823/flate2-1.0.10/src/lib.rs:89:1
   |
89 | extern crate libc;
   | ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/27812

error[E0658]: use of unstable library feature 'rustc_private': this crate is being loaded from the sysroot, an unstable location; did you mean to load this crate from crates.io via `Cargo.toml` instead?
  --> /home/rust/.cargo/registry/src/github.com-1ecc6299db9ec823/flate2-1.0.10/src/ffi.rs:52:20
   |
52 |     pub use libc::{c_int, c_uint, c_void, size_t};
   |                    ^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/27812

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,7 @@ streams.
 members = ['systest']
 
 [dependencies]
-libc = { version = "0.2", optional = true }
+libc = "0.2"
 miniz-sys = { path = "miniz-sys", version = "0.1.11", optional = true }
 libz-sys = { version = "1.0", optional = true }
 tokio-io = { version = "0.1.11", optional = true }
@@ -40,8 +40,8 @@ tokio-threadpool = "0.1.10"
 futures = "0.1"
 
 [features]
-default = ["miniz-sys", "libc"]
-zlib = ["libz-sys", "libc"]
+default = ["miniz-sys"]
+zlib = ["libz-sys"]
 rust_backend = ["miniz_oxide"]
 tokio = ["tokio-io", "futures"]
 

